<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSQLite3MultipleCiphersDemoClass</class>
 <widget class="QMainWindow" name="QSQLite3MultipleCiphersDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSQLite3MultipleCiphersDemo</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="createDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Create database</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TestDatabase.db</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePasswordButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kdf_algorithm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>kdf_algorithm:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kdf_iter">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>256000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>kdf_iter:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cipher">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>aes128cbc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>aes256cbc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>chacha20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sqlcipher</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rc4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ascon128</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cipher:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePasswordButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QSQLite3MultipleCiphersDemo.qrc"/>
 </resources>
 <connections/>
</ui>
